### DISCUSSION TITLE
The number of repositories is different the shown numbers

### DISCUSSION THREAD

**Discussion by CryoSky** [community/community#49385] [Repositories]
### Select Topic Area

Bug

### Body

I found there needs to be more consistency. If you check the number of repositories in my public profile https://github.com/CryoSky?tab=repositories, it shows 10. But the total count of shown repositories is only 9. I want to know why there is a hidden repository count. If I check the number in my own view, it shows 14, but I only found 13 repositories. Could anyone explain? Many thanks!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @CryoSky ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
